docs/building/internals.md

```diff
@@ -197,3 +197,67 @@ The layout of files should look like this:
 ```
 
 Lastly, run `nuget pack .` from within the folder. Note that you may need the latest `nuget.exe`.
+
+### PowerShell.Core.Instrumentation
+
+To successfully decode PowerShell Core ETW events, The manifest and resource binary need to be registered on the system.
+
+To create a new NuGet package for `PowerShell.Core.Instrumentation.dll`, you will need the PowerShell.Core.Instrumentation.nuspec found in the repo under src\PowerShell.Core.Instrumentation.
```

Comment:
PowerShell.Core.Instrumentation.nuspec should be put in the code block `...`

Reply:
Fixed